tests/pr_test.go

```diff
@@ -27,17 +33,61 @@ func TestRunDefaultExample(t *testing.T) {
 	assert.NotNil(t, output, "Expected some output")
 }
 
+func TestRunAutoscaleExample(t *testing.T) {
+	t.Parallel()
+
+	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+		Testing:            t,
+		TerraformDir:       autoscalingExampleTerraformDir,
+		Prefix:             "mongodb",
```

Comment:
Probably add a suffix to this so that we know which unit test the resource came from, see how the upgrade adds a "-upg" to the end, something like that.